client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -776,6 +783,10 @@ Params putParam(String key, TimeValue value) {
             return this;
         }
 
+        Map<String, String> getParam(){
```

Comment:
It would be better for this name to be something like `asMap()` - just from the name, I would assume `getParam` is the reverse of `putParam`, but that's not the case.